translations/en-us.yml

```diff
@@ -1160,6 +1160,12 @@ preprints:
     discover:
         title: 'Search'
     title: 'Preprints'
+    select:
+        page_title: 'Select Providers'
+        title: 'New Preprints'
+        heading: 'Select a preprint service'
+        paragraph: 'A preprint is a version of a scholarly or scientific paper that is posted online before it has undergone formal peer review and published in a scientific journal. <a href={link}>Learn More</a>.'
+        create_button: 'Create Preprint'
```

Comment:
I know this is a page that shows all of the preprint providers, but I would run this past Product team members to see if the usage of "Preprint" here is ok. Some providers don't call it a "Preprint", so just want to make sure they are in the loop about this language

Reply:
Good point! I am not sure how to accommodate for all word. Could be a New Submission", but it will be Product's call. I will check with Product.

Previous discussion with @bp-cos is that the header on this page will remain unbranded / static. One thought I had was to change it to branded when user selects the institution, which could be an alternative (but more complex improvement).

Reply:
Update: per Mark's comment, we will stick with "preprint(s)" on these non-branding pages.